### ISSUE TITLE
Scanning of VST's

### ISSUE THREAD

**Issue by curlymorphic** [LMMS/lmms#1837]
Currently we do very little to help our users when it comes to loading vst's. In Vestige, you are simply presented with a file dialog
![image](https://cloud.githubusercontent.com/assets/7412852/6546019/fb97d5d8-c599-11e4-97e0-05b62c3bcfff.png)
When opening as an effect, it is a little better, but really just a list of dll's
![image](https://cloud.githubusercontent.com/assets/7412852/6546028/292a599e-c59a-11e4-8d35-1ea9a2fd63ee.png)
This is prone to error, Instrument and effects are mixed together, but only one type is usable, there could be midi plugins, or the dll's could not be vst's.

It is common practice for Daws to scan vst folders. cataloging and deciding where these can be used.
The user is then presented with a list of suitable choices. When loading effects, the list of vst's would be filtered, and a similar styled list presented in vestige.

**Comment by Em-Lopez** [comment]
ALSO, on Windows 10, it seems like LMMS does not search for VSTs recursively.
For example, in a folder structure like this:
VSTs/
VSTs/MidSideMangler.dll
VSTs/G-Clip/
VSTs/G-Clip/GClip.dll
MidSideMangler will load, but GClip will not load because it is in subfolder.
When I changed it to this:
VSTs/
VSTs/MidSideMangler/
VSTs/MidSideMangler/MidSideMangler.dll
VSTs/GClip.dll
GClip did load, but MidSideMangler did not!
This demonstrates that LMMS does not have the capability to search recursively through folders for VSTs. Which kinda sucks.

I was able to work around this by making symlinks! But alas, that is very bad...
